app/assets/javascripts/init-elements.js

```diff
@@ -1,3 +1,65 @@
+this.Facebook = (function() {
+  var eventsBound, rootElement;
+
+  function Facebook() {}
+
+  rootElement = null;
+
+  eventsBound = false;
+
+  Facebook.load = function() {

```

Comment:
It looks like we are manually creating the facebook api scripts.  Is there a better way to do this?  Can't we mark the scripts already in the dom so that turbolinks knows to reload them?